### DISCUSSION TITLE
Linux vDisk Boot Plugin: grub-pc-bin_xxx.deb files?

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1375] [Q&A] [answered]
For the Linux vDisk Boot Plugin, where do I get the grub-pc-bin_xxx.deb files? I could not find a download link on the Linux vDisk Boot Plugin page and they are not found inside the vtoyboot ISO file.

https://www.ventoy.net/en/plugin_vtoyboot.html

**Comment by ventoy** [comment, marked as answer]
`grub-pc-bin_xxx.deb` is inside the linux distro ISO file.
For example, you test with `kali-linux-2020.1-installer-amd64.iso` then the deb file is in 
`kali-linux-2020.1-installer-amd64/pool/main/g/grub2/grub-pc-bin_2.04-5kali1_amd64.deb`
